app/components/shared/Slider.vue.ts

```diff
@@ -27,7 +26,11 @@ export default class SliderInput extends Vue {
     }, 500);
   }
 
-  @throttle(500)
+  sliderUpdate(value: number) {
+    this.$refs.inputbox.value = value;
```

Comment:
We really shouldn't need to use refs for this.  Can you explain to me what this is supposed to be doing?

Reply:
Input box value doesn't seem to get updated until the input event is handled by the parent component. Since the input event handler is delayed, it doesn't seem get updated along with the slider.

Reply:
Ah ok got it.  I think we should use an intermediate local state variable for that.  I think that should be handled by `SliderInput.vue.ts`  since it is the OBS-specific component doing the debouncing.  Mind if I push a commit to this branch?  Will be easier than typing it all out here.